src/js/expansionPanel.directive.js

```diff
@@ -102,15 +102,16 @@ function expansionPanelDirective() {
     vm.onRemove = onRemove;
     vm.init = init;
 
-    $attrs.$observe('disabled', function(disabled) {
-      isDisabled = (typeof disabled === 'string' && disabled !== 'false') ? true : false;
+	isDisabled = (typeof $attrs.disabled === 'string' && $attrs.disabled !== 'false') ? true : false;
 
-      if (isDisabled === true) {
-        $element.attr('tabindex', '-1');
-      } else {
-        $element.attr('tabindex', '0');
-      }
-    });
+	$element.attr('tabindex', isDisabled ? -1 : 0);
```

Comment:
the watcher will fire on startup so all you should need is a `if statement` wrapping the watcher. We do not want the watcher running if no `ngDisabled` attr is present. You should be able to get rid of lines 105 and 107